tests/neg-custom-args/fatal-warnings/i4008.scala

```diff
@@ -0,0 +1,48 @@
+// ===== Template annotations =====
+
+
+// class, 1TP, invalid ref
+@annotation.implicitNotFound("An implicit ShouldWarn1[${B}] is not in scope") // error
+class ShouldWarn1[A]
+
+// trait, 1TP, invalid ref
+@annotation.implicitNotFound("An implicit ShouldWarn2[${A}] is not in scope") // error
+trait ShouldWarn2[B]
+
+// trait, 2TP, 1 invalid ref
+@annotation.implicitNotFound("An implicit ShouldWarn3[${A},${B}] is not in scope") // error
+trait ShouldWarn3[B, C]
+
+// class, 2TP, 2 invalid refs
+// @annotation.implicitNotFound("An implicit ShouldWarn4[${A},${B}] is not in scope")
```

Comment:
This line yields 2 warnings, does anyone know how I would annotate this with the `// error` tag?
With one `// error`, the `testCompile` task tells me that more warnings were issued than expected.

Reply:
It should work with `// error // error` at the end of the line